eks/remote_state.tf.template

```diff
@@ -0,0 +1,8 @@
+terraform {
+  backend "s3" {
+    bucket  = "<basename>-terraform-state"  # replace with basename from terraform.tfvars
```

Comment:
Any specific reason you are not referencing the variables directly here?

Reply:
tried that first, but turns out you can’t use variable there

Reply:
Egh, that's disappointing. Alright then.